README.md

```diff
@@ -70,6 +70,8 @@ console.log(jpegImageData);
   height: 180,
   data: <Buffer 5b 40 29 ff 59 3e 29 ff 54 3c 26 ff 55 3a 27 ff 5a 3e 2f ff 5c 3c 31 ff 58 35 2d ff 5b 36 2f ff 55 35 32 ff 5a 3a 37 ff 54 36 32 ff 4b 32 2c ff 4b 36 ... > }
 */
+// write to file
+fs.writeFile("image.jpg",jpegImageData.data,'base64');
```

Comment:
```suggestion
fs.writeFileSync("image.jpg", jpegImageData.data);
```

encoding is ignored when data is a buffer, also we should either signify the callback if using the async version or just go with sync